naming/README.md

```diff
@@ -0,0 +1,76 @@
+
+# coreboot
+LinuxBIOS became `coreboot` in 2008. It is effectively the same project that evolved over time. `coreboot` (spelled lowercase) is a complete open source system firmware package, aimed at replacing proprietary implementations. It's also one of the most mature and well-maintained open source firmware projects.
+
+`coreboot` supports a **wide varieties of platforms**, and has a **modular architecture**. It provides **platform initialization** (CPU, DRAM, PCI, ACPI, SMBIOS, etc), **a filesystem** (CBFS) suitable for on-firmware storage, integration with **vendor extensions and blobs**, a friendly license, and a **wide variety of payloads**.
+
+This modular design also **enables various bootloader scenarios**, through coreboot payloads. Among the various options, there are:
+* *SeaBIOS*, a very popular open source BIOS implementation
+* *depthcharge*, the blazing-fast boot payload for ChromeOS/Chromebooks
+* *LinuxBoot*, of course! More details below
+* *UEFI*, via the open-source EDK II.
+
+# LinuxBoot
+LinuxBoot is not a product, but rather a **concept**. It's the idea of **booting Linux (OS) with Linux (system firmware)**. In a way, the same concept pioneered by LinuxBIOS.
+
+"LinuxBoot" is also often used as an umbrella name at Facebook to indicate how we do open source firmware, i.e. coreboot + Linux + u-root + systemboot. **Imagine it like a Linux distribution, but for firmware**. It is a collection of various open source components, glued together to work as a consistent firmware OS.
+
+Depending on who you are talking to, you may hear "LinuxBoot" used as a reference to "stripped UEFI, plus Linux". This is because LinuxBoot, when originally created, was meant to run a Linux kernel on top of a stripped UEFI firmware, not on coreboot. See also "NERF" below.
+
+# NERF
+This is the original name for the stripped UEFI, plus Linux, plus u-root. The name stands for Non-Extensible Reduced Firmware, as opposed to UEFI's Unified Extensible Firmware Interface. Basically, saying that NERF is an UEFI replacement that prefers to be more compact, less extensible, and a bit more opinionated. While extensibility is nice and often desirable, too much extensibility and too many "yes" can make a complex project very hard to maintain and keep secure.
+
+NERF started from Ron Minnich (one of the coreboot founders) at Google, and is now developed by a few other folks that are now part of Google's "NERF team". This name, in my understanding, is mostly used within Google, while "LinuxBoot" is becoming a more common name for this effort.
```

Comment:
`in my understanding,`: this should be made a certitude by the process of review, I don't think leaving unclear statements, will help our users.
So please Google's "NERF team" help us understand your point of view.

For ITRENEW we use the stripped UEFI, plus Linux, plus u-root solution and we tend to name it LinuxBoot. (We should probably brand it "Sesame LinuxBoot" or something like that).

Reply:
good point. Again the reason (as explained below) is that this was originally an internal post that I am making public after removing the parts that are not relevant to the public and trying to make it objective rather than subjective. Again, some of these slipped through the cracks :)

Reply:
re. branding Sesame LinuxBoot, I'd rather stick with names that hint about the technical solution rather than the author of that decision. If you start using "Sesame LinuxBoot" to refer to "stripped UEFI + Linux + u-root", and at some point you decide to change one of these components, that'll create another naming problem